### ISSUE TITLE
Harvard Dataverse Instance needs a BibTeX citation for citing the  Harvard Repository

### ISSUE THREAD

**Issue by sbarbosadataverse** [IQSS/dataverse.harvard.edu#129]
This BibTeX citation would acknowledge the Harvard Dataverse itself and not just a dataset.

Sample of what a BibTeX citation should contain attached

![image](https://user-images.githubusercontent.com/8322346/138157343-3242887d-71a8-4ac2-a043-71fc7d0a62f8.png)

LTS has a "URN" for us but not sure we need to have MIT attached to this URN anymore?  @lenwiz
 This is what the lawyer needs to help us determine:

https://nrs.harvard.edu/urn-3:hul.eresource:harvmitd" 

Also, note that the HOLLIS URN needs updating @jggautier :

"This is the URN used in HOLLIS:
http://id.lib.harvard.edu/alma/990121255650203941/catalog" ---->This points
to the software project, where the name is out of date by the way. It's no
longer The Dataverse Network. It's The Dataverse Project.

**Comment by sbarbosadataverse** [comment]
@pdurbin No, they wanted to reference "Harvard Dataverse" the actual repository, in the RT ticket they sent into support.

**Comment by jggautier** [comment]
I haven't received a reply about changing the name of the record in HOLLIS for the Dataverse Project. I sent another email to ask about changing the name. See the email in RT at https://help.hmdc.harvard.edu/Ticket/Display.html?id=366149

**Comment by jggautier** [comment]
There's more recent discussion in the email thread at https://help.hmdc.harvard.edu/Ticket/Display.html?id=366149.

I proposed that the HOLLIS record, https://github.com/IQSS/dataverse.harvard.edu/issues/129#issue-1031752334, continue to point to Harvard Dataverse, since I think it was always meant to represent the Harvard Dataverse Repository and not the Dataverse Project. The link points to Harvard Dataverse, although @sbarbosadataverse [you wrote](https://github.com/IQSS/dataverse.harvard.edu/issues/129#issue-1031752334) that it "points to the software project". And the record type is "database", which seems more like what Harvard Dataverse is and less like what the Dataverse Project is.

And I've proposed that the library folks help us create a new record about the Dataverse Project.

I asked the person at the library helping us if this makes sense and that I'd follow up with you @sbarbosadataverse.

Let me know what you think.

**Comment by jggautier** [comment]
In that email with @sbarbosadataverse and our friend from Harvard Library that's helping, I wrote that @sbarbosadataverse agreed with what we proposed, and re-listed that proposal:

- Change the title of the record at https://id.lib.harvard.edu/alma/990121255650203941/catalog to "Harvard Dataverse Repository"
- Remove the variant title "Dataverse Network" from that record (https://id.lib.harvard.edu/alma/990121255650203941/catalog)
- Create a new record about the Dataverse Project. That record would include a link that points to the project's website, https://dataverse.org

I also asked if the new record, about the Dataverse Project, should include the variant titles "Dataverse Network" and "IQSS Dataverse Network". They asked about variant titles earlier in the conversation so I think need to consider how records in HOLLIS have been found before changes were made to them.

**Comment by pdurbin** [comment]
Just a thought for a future feature of Dataverse: have it generate a citation for itself in various formats, including BibTeX.

**Comment by jggautier** [comment]
Oh cool!

I imagine that after the records are updated and created in HOLLIS, so that there's one record about Harvard Dataverse the service, and another about The Dataverse Project, I would remove "and update name of Dataverse Project in HOLLIS" from this GitHub issue's title.

Then @sbarbosadataverse would follow up on next steps. Is that right?

**Comment by pdurbin** [comment]
@siacus and I just discussed how to cite the Dataverse software and we decided to use Gary's 2007 paper:

King, G. (2007). An Introduction to the Dataverse Network as an Infrastructure for Data Sharing. Sociological Methods & Research, 36(2), 173-199. https://doi.org/10.1177/0049124107306660

I just updated https://dataverse.org/about which previously linked to https://gking.harvard.edu/files/abs/dvn-abs.shtml

As I wrote at https://github.com/IQSS/dataverse/issues/10187#issuecomment-2501682857 here's how the "about" page looks now:

![Screenshot 2024-11-26 at 1 43 17 PM](https://github.com/user-attachments/assets/ef2ca96a-1911-4b4a-8e89-46d57f846872)

**Comment by pdurbin** [comment]
Maybe under "Notes" we could add something like "Please use https://doi.org/10.1177/0049124107306660 to cite." 🤷 

![Screenshot 2024-12-16 at 2 37 19 PM](https://github.com/user-attachments/assets/31518f81-8376-4fc0-9b91-fc95194d4ab3)

But yeah, some people might go for that bit "citation" button and not notice a lil' note down below.

**Comment by sbarbosadataverse** [comment]
@jggautier Hey Julian, where are we with this issue? Is it complete?

**Comment by jggautier** [comment]
Hey @sbarbosadataverse. It's not complete. I [suggested BibText text](https://github.com/IQSS/dataverse.harvard.edu/issues/129#issuecomment-3571346353) that people might use for citing Harvard Dataverse, you asked if we could add the BibTeX citation to the Harvard Dataverse support site at https://support.dataverse.harvard.edu, and I chatted with @dliburd about that and [suggested](https://github.com/IQSS/dataverse.harvard.edu/issues/129#issuecomment-3628059525) the site's About page.

Maybe this is dependent on a larger redesign of the Harvard Dataverse support site that your working on Dwayne? Os that fair to say?

I see a GitHub issue at https://github.com/IQSS/dataverse.harvard.edu/issues/67 about making changes to the site, but maybe more recent work is being tracked someplace else?
